src/pages/patientView/PatientViewPage.tsx

```diff
@@ -304,7 +304,7 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
                                         sampleLabels={sampleManager.sampleLabels}
                                         sampleColors={sampleManager.sampleColors}
                                         sampleManager={sampleManager}
-                                        getWidth={()=>$(window).width()-140}
+                                        getWidth={()=>$(window).width() - ((patientViewPageStore.mutationData.result.length > 0) ? 140 : 40)}
```

Comment:
Yeah, I like that.  getContainerWidth.   and you're right the adjustment belongs inside because it's intrinsic information.  will do.